io.gitlab.irrlamb.yml

```
@@ -0,0 +1,23 @@
+app-id: io.gitlab.irrlamb
```

```suggestion
app-id: io.gitlab.jazztickets.irrlamb
```

Excactly like with github

---

The new webpage is https://irrlamb.gitlab.io and https://gitlab.com/irrlamb. Shorter is better.

---

but you don't own `gitlab.io`, so an acceptable app-id would be `io.gitlab.irrlamb.irrlamb`

---

Seems pretty silly to me, but whatever i'll just patch it in to be jazztickets

---

it's not silly. It's even in the documentation.